src/features/import/utils/prepareImportOperation.ts

```diff
@@ -0,0 +1,63 @@
+import { CellData, ColumnKind, Sheet } from './types';
+
+export type OpsType = {
+  fields?: Record<string, CellData>;
+  op: 'person.import';
+  organizations?: number | null;
+  tags?: number[];
+};
+export interface ImportOpsProp {
```

Comment:
I think the name of this is confusing me a bit. The name sounds as if it is a type of props to a component, but it's used as the type for what this function returns... could ImportOperations be a better name?

Reply:
I always add `Prop` as suffix at the end of `interface`. Thought that it is a pattern in our code base?? 🤷 No idea.. Could add `I` as prefix as well

Reply:
The `Prop` prefix, or really `Props` (plural) is what we use when the interface/type defines the properties (props) for a component. It's not something we _always_ use for _all_ interfaces/types.

We also don't use the `I` prefix, as far as I know, in this codebase. I've used it elsewhere, but I don't think it's used here.

Thinking about it now, I kind of think this function should really just return a list of ops, not an object containing the list of ops, unless you see some reason why I'm wrong about this. So the return value for the function should be `OpsType[]`.

Although I also think the name `OpsType` is not great. Maybe it could be `ZetkinPersonImportOp` instead, to indicate that it's related to the Zetkin API, and that it's a single operation of type `person.import`?

In summary, I think you should:
* Remove the `ImportOpsProp` entirely
* Rename `OpsType` to `ZetkinPersonImportOp`
* Refactor `prepareImportOperations` to return an array of operations instead

Reply:
Thanks for feedback @ziggabyte and @richardolsson ! I should have thought about naming things more carefully. Now I know so will try to be better next time 👍